packages/block-editor/src/components/resolution-tool/index.js

```diff
@@ -44,6 +44,7 @@ export default function ResolutionTool( {
 			panelId={ panelId }
 		>
 			<SelectControl
+				__nextHasNoMarginBottom
```

Comment:
Doesn't matter because this is wrapped in a `ToolsPanelItem`, which already does normalization on `BaseControl` margins.